cmd/configure.go

```diff
@@ -78,6 +89,16 @@ var configureTargetCmd = &model.ExecutableCommand[ConfigureTargetFlags]{
 	},
 }
 
+type ConfigureGithubFlags struct{}
+
+var configureGithubCmd = &model.ExecutableCommand[ConfigureGithubFlags]{
+	Usage:        "github",
+	Short:        "Configure your speakeasy workflow to generate and publish from your github repo.",
```

Comment:
Should probably capitalize Speakeasy. Also we should make a better distinction between `short` and `long`